### PR TITLE
quicklogic: add support to generate bitstream with the toolchain

### PR DESCRIPTION
**Pull request by acomodi**
Signed-off-by: Alessandro Comodi <acomodi@antmicro.com>

This PR adds support to write the bitstream and FASM to the installed toolchain.

### CODE REVIEW

**Review by kkumar23 — COMMENTED** [review]
except the top module counter name, LGTM

File: quicklogic/qlf_k4n8/tests/counter_16bit/counter_16bit.v [hunk 1]
```diff
@@ -6,7 +6,7 @@
 //
 // Version 1.0 : Initial Creation
 //
-module top (clk, reset, enable, count);
+module counter_16bit (clk, reset, enable, count);
```
[the remark this answers is not in the document]
Comment (acomodi) [reply quicklogic/qlf_k4n8/tests/counter_16bit/counter_16bit.v:9]: > Any reason to change the topmodule name?

Not really, it was just to keep consistency with the other tests. I may revert this to be `top` if it is preferred.

[the remark this answers is not in the document]
Comment (kkumar23) [reply quicklogic/qlf_k4n8/tests/counter_16bit/counter_16bit.v:9]: @acomodi , yes you can revert the name, and then its good to merge

[the remark this answers is not in the document]
Comment (acomodi) [reply quicklogic/qlf_k4n8/tests/counter_16bit/counter_16bit.v]: All right, done.

File: quicklogic/common/toolchain_wrappers/ql_symbiflow [hunk 2]
```diff
@@ -470,13 +476,8 @@ elif [[ ! -z "$OUT" && $1 == "-compile" ]];then
...
-  # FIXME: No bitstream for qlf_k4n8
-	  if [ "${FAMILY}" == "qlf_k4n8" ]; then
-    		cd $SOURCE;make -f Makefile.symbiflow ${BUILDDIR}/${TOP_FINAL}.route  || exit
-  	  else
-    		cd $SOURCE;make -f Makefile.symbiflow ${BUILDDIR}/${TOP_FINAL}.bit  || exit
-  	  fi
+  echo -e "Running Synth->Pack->Place->Route->FASM->bitstream"
+  	cd $SOURCE;make -f Makefile.symbiflow ${BUILDDIR}/${TOP}.bit  || exit
   fi
 fi
```
[the remark this answers is not in the document]
Comment (acomodi) [reply quicklogic/common/toolchain_wrappers/ql_symbiflow:483]: Changed the `all` target to run bitstream generation

**Comment by kkumar23** [comment]
you can merged once CI is green.

**Pull request merged**